modules/launcher/src/test/java/org/mule/runtime/module/launcher/DeploymentServiceTestCase.java

```diff
@@ -1294,6 +1302,78 @@ public void deploysAppZipWithPlugin() throws Exception
     }
 
     @Test
+    public void deploysAppZipWithContainerPluginBroken() throws Exception
```

Comment:
Actually there are unit tests for this but here I wanted to do a functional test that uses an application that would get access to classes from a core application plugin.